DDG4/src/Geant4StepHandler.cpp

```diff
@@ -125,6 +125,7 @@ double Geant4StepHandler::birkAttenuation() const    {
   const G4Track* trk = step->GetTrack();
   const G4ParticleDefinition* particle = trk->GetDefinition();
   const G4MaterialCutsCouple* couple = trk->GetMaterialCutsCouple();
+  s_emSaturation.SetVerbose(true);
```

Comment:
I have updated it as you suggested.
Should I keep this pull request as the current?
Or should I remove this pull request, and generate a new one to include the last update only?